src/app/uploads/application/dtos/upload-avatar.dto.ts

```diff
@@ -0,0 +1,6 @@
+import { ApiProperty } from '@nestjs/swagger';
+
+export class UploadAvatarDto {
```

Comment:
```suggestion
export class UploadImageRequestDto {
```
عشان نخليها مناسبة لاي صورة بشكل عام